src/test/java/address/unittests/storage/StorageManagerTest.java

```diff
@@ -62,6 +76,20 @@ public void setup(){
     }
 
     @Test
+    public void start_correspondingMethodCalled() throws Exception {
+        storageManagerSpy.start();
+        PowerMockito.verifyPrivate(storageManagerSpy).invoke("loadDataFromFile", userPrefsMock.getSaveLocation());
+    }
+
+    @Test
+    public void saveAddressBook_noException() throws IOException, DataConversionException {
+        StorageManager.saveAddressBook(DUMMY_DATA_FILE, EMPTY_ADDRESSBOOK);
+
+        PowerMockito.verifyStatic();
+        FileUtil.createIfMissing(DUMMY_DATA_FILE);
+    }
```

Comment:
If a method is not tested, create an empty test method and explain why it is not tested as a comment inside it.